layouts/calendar_layout.py

```diff
@@ -0,0 +1,81 @@
+import dash_core_components as dcc
+import dash_html_components as html
+import layouts.style.style as style
+import calendar
+from tools.calendar_generation import generate_month
+from tools.calendar_generation import print_title
+
+CAL_BUT = [
+        [
+           'Previous'
+	],
+        [
+           'Next'
+        ]
```

Comment:
Why does it have to be nested list with only one element? Wouldn't it be better to just specify ['Previous', 'Next']?